### ISSUE TITLE
It makes NetLogo "not responding"

### ISSUE THREAD

**Issue by MasterDoo** [mbi2gs/netlogo-matlab-extension#4]
"On Mac with Sierra, Matlab 2016a, NetLogo 5.3.1"
After typing "extensions [matlab]" in the code of netlogo and clicking "check" Netlogo became "Not responding."

**Comment by MasterDoo** [comment]
Also, the M-file in the zip downloaded was named "MatlabServer1.m" with a "1". In accordance to the instruction, it should be "matlabserver.m"

**Comment by mbi2gs** [comment]
Hi MasterDoo, thanks for your comments. On the first issue, there have been other people who have had trouble getting it to work on newer Mac OS's. I don't have access to a new Mac right now, so unfortunately that issue will have to wait. I did recently update it to keep things working on Windows, so if you are able to work on a PC, I would recommend doing that for now.

I updated the instructions to say "matlabServer1.m". Thanks for pointing that out.
